src/index.pug

```diff
@@ -0,0 +1,15 @@
+doctype html
+html(lang='en')
+  head
+    title title here
+    meta(charset='UTF-8')
+    meta(name='viewport' content='width=device-width, initial-scale=1')
+    link(href='/css/vendor.css' rel='stylesheet')
+    link(href='/css/application.css' rel='stylesheet')
+  body
```

Comment:
Use HTML5 tags instead of classes

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header